cfgov/jinja2/v1/_includes/atoms/partially-styled-link.html

```diff
@@ -0,0 +1,29 @@
+{# ==========================================================================
+
+   Partially styled link
+
+   ==========================================================================
+
+   Description:
+
+   Create a partially underlined link atom when given:
+
+   value:         An object with the following options for value.
+
+   value.url:     A string for the URL of the link.
+
+   value.unstyled:    A string for the text of the link.
+
+   value.styled: A string for the underlined text of the link.
+
+   ========================================================================== #}
+
+{% macro render(value) -%}
+
+<a href="{{ value.url }}" class="a_link a-link__partially-styled"> 
+	<span class="a-link__partially-styled_unstyled">{{ value.unstyled|safe }} </span>
```

Comment:
The name `a-link__partially-styled_unstyled` is a bit strange to me.  It's styled but it's not styled.